### PR TITLE
A little cleanup, new example

### PR DESCRIPTION
**Pull request by PTS93**
I edited all composite examples that use the mouse to behave the same as the main hid_mouse example.
I move the ramdisk.h to the src directory so it does not have to be copied to every example that uses it.

This should also avoid confusion with users who write a new sketch and wonder where ramdisk.h comes from.

I also added a new composite example using raw hid and updated the node.js example to the one already present in the main tinyusb repo. I also added a readme for new users to the example folder.

### CODE REVIEW

**Comment by PTS93** [comment]
I think using the ramdisk would actually be used quite a bit on cheaper boards like the Trinket m0 for small stuff like config files.
Would it make sense to create an example folder in the src folder where ramdisk.h would be placed instead, making clear it is not part of the main source code and not needed by the library itself?

If you feel that your 'hack' is not ready for production use maybe it make sense to make it a library like SDFat and fix whatever you think is missing? I do think its very valuable.

**Comment by hathach** [comment]
> I think using the ramdisk would actually be used quite a bit on cheaper boards like the Trinket m0 for small stuff like config files.
> Would it make sense to create an example folder in the src folder where ramdisk.h would be placed instead, making clear it is not part of the main source code and not needed by the library itself?
> Just for the sake of maintaining it seems error prone to have the same code duplicated over multiple directories.
> 
> If you feel that your 'hack' is not ready for production use maybe it make sense to make it a library like SDFat and fix whatever you think is missing? I do think its very valuable.

It is not a part of library sources which is about USB class drivers, and should be stayed in the application where it is used. It is a clever hack, though even if it is error-free, it won't be placed in the `src` of this library. User can create a higher library (middle ware above usb) to manipulate the ramdisk if needed.

**Comment by PTS93** [comment]
Ok fair enough :)
I reversed moving ramdisk.h with the new commit

File: examples/MassStorage/msc_ramdisk_dual/ramdisk.h [hunk 1]
```diff
@@ -25,30 +25,25 @@
...
-//--------------------------------------------------------------------+
-#define README0_CONTENTS \
-"LUN0: This is tinyusb's MassStorage Class demo.\r\n\r\n\
-If you find any bugs or get any questions, feel free to file an\r\n\
-issue at github.com/hathach/tinyusb"
+#define README_CONTENTS \
+"This is Adafruit TinyUSB MassStorage device demo on RAM disk."
 
-uint8_t msc_disk0[DISK_BLOCK_NUM][DISK_BLOCK_SIZE] =
+uint8_t msc_disk[DISK_BLOCK_NUM][DISK_BLOCK_SIZE] =
```
Comment (hathach) [inline examples/MassStorage/msc_ramdisk_dual/ramdisk.h]: please revert to the previous version of ramdisk.h in this dual lun example. It is intended to show up as `TinyUSB 0` and `TinyUSB 1` with different readme file contents as well

**Comment by PTS93** [comment]
Ooh right I misunderstood this before, sorry for the hassle. Should be fine now.

**Comment by PTS93** [comment]
Is it normal that the Branch check takes so long?

**Comment by ladyada** [comment]
hmm try adding a commit like a whitespace change, see if that kicks it?

**Comment by PTS93** [comment]
Could be that we are dealing with this issue?
https://travis-ci.community/t/travis-ci-branch-expected-waiting-for-status-to-be-reported/829

I don't fully understand whats going on there but seems this is an issue with branch check and forks that don't have CI set up?

Also not sure how to resolve this requested change? I did resolve the conversation about it after committing the fix.

**Comment by ladyada** [comment]
ok i enable/disabled it and now its ok!

**Comment by PTS93** [comment]
Great! Hopefully more to come :)

**Pull request merged**